docker/exercise1/main.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+import numpy as np
+from PIL import Image
+
+args = sys.argv
+
+exec = os.environ.get('EXEC')
+
+if exec == 'true':
+    a = np.array([1, 2, 3, 4, 5, 6])
+
+    b = a.reshape((2, 3))
+
+    print(b)
+
+    image = Image.open('image.png')
+
+    width = int(args[1])
```

Comment:
Will this work if the args[1] is a string value?
What wil happen if only one argument is passed?
Worth adding a check for number of args required